README.md

```diff
@@ -789,6 +789,57 @@ factory.get(config).then(model => {
 | config.version | String | Yes | Version of the template |
 | config.label | String | No | Label of the template |
 
+### Template Tag Model
+```js
+'use strict';
+const Model = require('screwdriver-models');
+const factory = Model.TemplateTagFactory.getInstance({
+    datastore
+});
+const config = {
+    name: 'testTemplate',
+    tag: 'stable',
+    version: '1.3'
+}
+
+factory.create(config)
+    .then(model => {    // do something
+    });
+```
+
+#### Update
+Update a specific template tag
+```js
+model.update()
```

Comment:
Showing that you're changing a model's value then calling update would be more valuable